modules/hub/src/OptimisticPaymentsService.ts

```diff
@@ -0,0 +1,114 @@
+import log from "./util/log";
+import { Poller } from "./vendor/connext/lib/poller/Poller";
+import ChannelsDao from "./dao/ChannelsDao";
+import DBEngine from "./DBEngine";
+import { convertChannelState, convertPayment, PurchasePayment } from "./vendor/connext/types";
+import OptimisticPaymentDao from "./dao/OptimisticPaymentDao";
+import { OptimisticPurchasePaymentRow } from "./domain/OptimisticPayment";
+import PaymentsService from "./PaymentsService";
+
+const LOG = log('OptimisticPaymentsService')
+
+const POLL_INTERVAL = 1000
```

Comment:
it affects the speed at which payments are resolved once collateral is in, so it would be a potential poller length interval of wait time on the ui, can turn it down though